src/xdp/xsk.c

```diff
@@ -1661,7 +1661,8 @@ XskNotifyRxQueue(
 NTSTATUS
 XskValidateDatapathHandle(
     _In_ HANDLE XskHandle,
-    _In_ XDP_RX_QUEUE *RxQueue
+    _In_ XDP_RX_QUEUE *RxQueue,
+    _In_ BOOLEAN IgnoreQueueMatch
```

Comment:
I'd just always ignore the queue match condition here.

Reply:
Done